build.sbt

```diff
@@ -1,14 +1,14 @@
 
-scalaVersion := "2.12.4"
+scalaVersion := "2.12.15"
 
-val sparkVersion = "2.4.0"
+val sparkVersion = "3.2.1"
```

Comment:
Given that [Spark 3.3 is already released](https://spark.apache.org/releases/spark-release-3-3-0.html) and this PR isn't merged, would it be worthwhile to upgrade to Spark`3.3.0`?